packages/riipen-ui-docs/src/pages/components/paper/Elevation.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+import Paper from "riipen-ui/components/Paper";
+
+export default function Elevation() {
+  const style = {
+    marginRight: "20px"
+  };
+
+  return (
+    <React.Fragment>
+      <Paper
+        style={{
+          height: "100px",
+          width: "100px"
+        }}
+      />
+
```

Comment:
We general tend to avoid blank lines like this in our React code. I wouldn't say one way is better than the other, but we should stay consistent with the rest of the code. Go ahead and remove these blank lines in this file and the outline file.